src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -56,6 +56,7 @@ void CLibLinear::init()
 	C2=1;
 	set_max_iterations();
 	epsilon=1e-5;
+	set_compute_bias(false);
```

Comment:
Can you investigate whether this is a problem or not?
If de-activated, liblinear should document this, and maybe even overload the setter and throw an error telling the user that this is not allowed